autoload/go/oracle.vim

```diff
@@ -143,6 +143,25 @@ func! s:RunOracle(mode, selected) range abort
     return out
 endfun
 
+function! go#oracle#Scope(...)
+	if len(a:000)
+        if len(a:000) == 1 && a:1 == '""'
```

Comment:
I never reviewed Vimscript before but I'm wondering about the indentation here...